iceoryx2/src/config.rs

```diff
@@ -394,10 +392,18 @@ impl Config {
     /// [`Config::setup_global_config_from_file()`]
     /// is called after this function was called, no file will be loaded since the global default
     /// config was already populated.
+
     pub fn global_config() -> &'static Config {
         if !ICEORYX2_CONFIG.is_initialized()
             && Config::setup_global_config_from_file(unsafe {
-                &FilePath::new_unchecked(DEFAULT_CONFIG_FILE)
+                &FilePath::new_unchecked(
```

Comment:
Using `new_unchecked` will panic if the path is too long: https://github.com/eclipse-iceoryx/iceoryx2/blob/main/iceoryx2-bb/container/src/byte_string.rs#L284

Since the path length depends on the environment it's not a guarantee this won't happen anymore. Would be better to catch this case, print a warning message, then use the default config.